deployment/development/docker-compose.yml

```diff
@@ -26,10 +26,13 @@ services:
       - postgres-data:/var/lib/postgresql/data
       - ../scripts/DB:/scripts
 
-    # Debugging
+  # Debugging database
+  adminer:
```

Comment:
this should have a `links` section to the db service, yeah?

Reply:
Links not needed.  You just connect to db.